.circleci/config.yml

```diff
@@ -1,18 +1,9 @@
 version: 2.1
 
 jobs:
-  style_check:
-    docker:
-      - image: gcr.io/rf-public-images/circlemator:latest
-    steps:
-      - checkout
-      - run:
-          name: Style check
-          command: circlemator style-check --base-branch=master
```

Comment:
This just calls Pronto: https://github.com/rainforestapp/circlemator/blob/8a6758bfe1ea38354b5edbd65d5b7451b7e9d6b5/lib/circlemator/code_analyser.rb#L20

So maybe we can just use the same Pronto action as main app?

Reply:
Oh, I see you did that so ✅